app/src/main/java/me/hufman/androidautoidrive/music/controllers/MusicAppController.kt

```diff
@@ -42,6 +44,10 @@ interface MusicAppController {
 
 	fun isShuffling(): Boolean
 
+	fun isRepeatingAll(): Boolean
```

Comment:
I think these two functions should be combined to a single function, such as `getRepeatMode()`. It could return a PlaybackStateCompat.REPEAT* constant int, but it could also return a new Enum to represent the tristate repeat mode. These two separate boolean state flags doesn't encode the same meaning.

Reply:
I agree it would be better as a single function, I added a new getRepeatMode method that returns a new RepeatMode enum that way it is decoupled from android, spotify, etc